tests/test_asset_store.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+from unittest.mock import MagicMock
```

Comment:
I was reading about MagicMock on friday :)

Reply:
It's aptly named, it really is magic. I'm still not entirely sure I understand it, but i can get it to work most of the time, and it's super handy...

Reply:
I was reading on Friday Mock vs MagicMock. Very interesting, but indeed needs some practice to really get the feeling for it. I get to know quite a bit from the pr reviews, i like it! ;-)